lib/suse/connect/api.rb

```diff
@@ -85,7 +85,7 @@ def make_paginated_request(method, url, options = {})
       private
 
       def system_uuid
-        @system_uuid ||= if File.exist?(UUID_FILE_LOCATION)
+        @system_uuid ||= if File.exist?(UUID_FILE_LOCATION) && !File.empty?(UUID_FILE_LOCATION)
```

Comment:
Maybe add or adapt the test for this line in `spec/suse/connect/api.spec`.

Reply:
_done!_ I've added a test, that creates an empty file and checks if the "test-uuid" is present. (like it is done with the existing tests)